lib/php/rendition-factory/src/DTO/BuildConfig/FamilyBuildConfig.php

```diff
@@ -14,7 +14,10 @@ public function __construct(
 
     public function getTransformations(): array
     {
-        return $this->transformations;
+        return array_filter(
+            $this->transformations,
+            fn (Transformation $transformation) => $transformation->isEnabled()
+        );
     }
```

Comment:
Why not filtering in the YamlLoader at early stage?

Reply:
yes, we may implement the `enable` as generic, to disable (remove ? hide ?) a whole block anywhere. Thus it will work for modules, filters, options, ... with no specific `if` in code

Reply:
Not anywhere, only on the transformation. But you can filter transformations at the parsing phase, so we will never forget to filter disabled transformations after that.

```php
    // in YamlLoader
    private function parseFamilyConfig(array $data): FamilyBuildConfig
    {
        if (empty($data['transformations'])) {
            throw new ModelException('Missing transformations');
        }
        $transformations = [];
        foreach ($data['transformations'] as $transformation) {
            if ($transformation['enabled'] ?? true) {
                $transformations[] = $this->parseTransformation($transformation);
            }
        }
        return new FamilyBuildConfig($transformations, $data['normalization'] ?? []);
    }
```